src/components/breadcrumb/BreadcrumbBasic.vue

```diff
@@ -26,14 +27,13 @@ export default {
 }
 </script>
 
-<style scoped lang="scss">
+<style lang="scss">
+@import "../../styles/theme";
```

Comment:
Importing this line for every component is looks like hassle to me. I think we need to think about some approach so that styling will be mapped automatically when someone address a variable from 'theme file'

Reply:
yeah I agree. But I'm happy adding for now so we can make the global scss work :)